app/assets/stylesheets/thredded/components/_topic-header.scss

```diff
@@ -60,3 +60,67 @@
   font-size: $thredded-font-size-small;
   color: $thredded-secondary-text-color;
 }
+
+
+@mixin following {
+  &::after {
+    display: inline-block;
+    font-size: 1em;
+    width: 1.4rem;
+    height: 1.4rem;
+    position: absolute;
+    top: 0;
+    right: -1.6rem;
+    background-image: image-url("thredded/follow.svg");
```

Comment:
@glebm Yes, this won't look good with other themes with other bg colors.

Didn't really understand inline svg before -- this is amazingly useful.

Just to summarize: I can define an svg inline in the dom (but not visible in this position -- will research this more carefully), and then add the `<use>` within the dom each place we want it to go, and then style that as needed for different themes. This will work well for my js-attachment needs too.